paper-progress.html

```diff
@@ -325,7 +329,10 @@
 
       this.secondaryProgress = secondaryProgress;
 
-      this.setAttribute('aria-valuenow', value);
+      if (!this.indeterminate) {
+        this.setAttribute('aria-valuenow', value);
+      }
+
       this.setAttribute('aria-valuemin', min);
       this.setAttribute('aria-valuemax', max);
```

Comment:
The spec seems to be ambiguous on this. Here's a bunch of other UI toolkits removing only aria-valuenow and not the rest --

- https://github.com/jquery/jquery-ui/blob/master/ui/widgets/progressbar.js#L148
- https://github.com/dojo/dijit/blob/master/ProgressBar.js#L106